### PR TITLE
makes brainwashing disk role restricted and debuffs its price

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
3tc back to 5tc
role restricted to medical doctor and roboticist once more

also before you ask no this technically isn't in response to the funny brainwash event that happened like 2 hours before this pr opened i was just notified about it and thought brainwashing disk for all traitors is bad anyways

## Why It's Good For The Game
infinite antag conversion? traitors are meant to be solo antags or in small groups, and their only other conversion methods are romerol (25tc and infecteds lose perks of being a non-simplemob) and emag borg (6tc for 15 or 7 uses, requires surgery/decapitation/science and borg bodies to put the brain into, and you have to emag the borg before it runs away from you)

role restricteds are allowed to be op though
5tc because roboticists and medical doctors both get it; this means it's VERY good for meddoctors (free access to sleepmixes already) and VERY challenging for roboticists (no free access to sleepmixes... until sci researches chem dispensers anyways), although roboticists have a very easy time converting with emag so it balances out somewhat

## Changelog
:cl:
balance: brainwashing disk has lost its cost buffs (3->5) and is role restricted once more (medical doctor/roboticist)
/:cl:

### CODE REVIEW

**Comment by lolman360** [comment]
although roboticists

**Comment by rossark** [comment]
Although roboticist WHAT?

**Comment by ghost** [comment]
Although roboticists

File: code/modules/uplink/uplink_items/uplink_devices.dm [hunk 1]
```diff
@@ -240,7 +240,8 @@
 	desc = "A disk containing the procedure to perform a brainwashing surgery, allowing you to implant an objective onto a target. \
 	Insert into an Operating Console to enable the procedure."
 	item = /obj/item/disk/surgery/brainwashing
-	cost = 3
+	restricted_roles = list("Medical Doctor", "Roboticist")
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items/uplink_devices.dm]: Can you place this stuff in the correct file and change the name path so it shows in "restricted roles" tab?

**Comment by timothyteakettle** [comment]
let paramedics/chemists buy it, it's a cool item and restricting its use sucks

File: code/modules/uplink/uplink_items/uplink_devices.dm (hunk 1 above)
Comment (ghost) [reply code/modules/uplink/uplink_items/uplink_devices.dm]: Thanks for the reminder

**Comment by Ludox235** [comment]
Do you ever do any pr that doesn't involve nerfing traitor items
Also I don't get why restricting it. Yeah sure make it 5tc but you still need to grab a target, subdue them long enough to do the surgery without them going ; help name traitor ahaaaaaa im being stune aaaa

**Comment by lolman360** [comment]
nameko balances for the 10%

**Comment by rossark** [comment]
I haven't tried it but I think you can brainwash dead bodies too and then just clone/defib them.

**Comment by ghost** [comment]
> Do you ever do any pr that doesn't involve nerfing traitor items
> Also I don't get why restricting it. Yeah sure make it 5tc but you still need to grab a target, subdue them long enough to do the surgery without them going ; help name traitor ahaaaaaa im being stune aaaa

sorry i'm not tupinambis

**Pull request merged**